lib/sdp/importers/spreadsheet.rb

```diff
@@ -194,7 +205,7 @@ class Spreadsheet
           data_type: 'Question Response Type (R)',
           tag_table: 'Question Tag Table (O)',
           value_set_table: 'Local Response Set Table (C)',
-          value_set: 'Response Set Source (C)'
+          value_set: 'Local Response Set Table (C)'
```

Comment:
It looks like this change is changing value_set to be the same as value_set_table (line 207 above). Should one of these be eliminated. If so the exporter needs to be updated as well in file `app/views/surveys/generic_spreadsheet.xlsx.axlsx` at line 98 and lines 33-34. Currently the exporter is creating the old convention and if the importer is updated to only look at one column the exporter should only create one column with the data expected.

Reply:
Yes- due to changes discusses with Kelly and Drew